plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -124,6 +127,24 @@ public IcebergFileWriter createFileWriter(
         }
     }
 
+    public IcebergFileWriter createPositionDeleteWriter(
+            Path outputPath,
+            Schema icebergSchema,
+            JobConf jobConf,
+            ConnectorSession session,
+            HdfsContext hdfsContext,
+            IcebergFileFormat fileFormat)
+    {
+        switch (fileFormat) {
+            case PARQUET:
+                return createParquetWriter(outputPath, icebergSchema, jobConf, session, hdfsContext);
+            case ORC:
+                return createOrcWriter(FULL_METRICS_CONFIG, outputPath, icebergSchema, jobConf, session);
+            default:
+                throw new TrinoException(NOT_SUPPORTED, "File format not supported for Iceberg: " + fileFormat);
```

Comment:
I think this is unclear. The format is not supported by Trino. But Avro, for example, is supported by Iceberg.

Reply:
Removed "for Iceberg".